extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/HttpConfiguration.java

```diff
@@ -203,6 +203,19 @@
     @ConfigItem
     public Map<String, SameSiteCookieConfig> sameSiteCookie;
 
+    /**
+     * If responses should be compressed.
+     *
+     * Note that this will attempt to compress all responses, to avoid compressing
+     * already compressed content (such as images) you need to set the following header:
+     * 
+     * Content-Encoding: identity
+     * 
+     * Which will tell vert.x not to encode the response.
```

Comment:
Is that a typo? Should it instead say "... not to compress the response."

Reply:
Fixed